templates/adding_a_new_model/cookiecutter-template-{{cookiecutter.modelname}}/modeling_tf_{{cookiecutter.lowercase_modelname}}.py

```diff
@@ -2356,6 +2447,25 @@ def call(
             encoder_hidden_states=inputs["encoder_outputs"].hidden_states,
             encoder_attentions=inputs["encoder_outputs"].attentions,
         )
+    
+    def serving_output(self, output):
+        return TFSeq2SeqModelOutput(
+            last_hidden_state=output.last_hidden_state,
+            past_key_values=tf.tuple(output.past_key_values)[1],
+            decoder_hidden_states=tf.convert_to_tensor(output.decoder_hidden_states)
+            if self.config.output_hidden_states
+            else None,
+            decoder_attentions=tf.convert_to_tensor(output.decoder_attentions)
+            if self.config.output_attentions
+            else None,
+            encoder_last_hidden_state=output.encoder_last_hidden_state,
+            encoder_hidden_states=tf.convert_to_tensor(output.encoder_hidden_states)
+            if self.config.output_hidden_states
+            else None,
+            encoder_attentions=tf.convert_to_tensor(output.decoder_attentions)
+            if self.config.output_attentions
+            else None,
```

Comment:
For those, we should do like BART and have the long lines as intermediate results before (on one line if possible).